tiledb/sm/fragment/fragment_info.h

```diff
@@ -107,6 +107,9 @@ class FragmentInfo {
   Status get_total_cell_num(uint64_t* cell_num) const;
 
   /** Retrieves the name of the fragment with the given index. */
+  const std::string& get_fragment_name_str(uint32_t fid) const;
```

Comment:
We should call this `fragment_name`. We don't need a `get_` prefix for functions that return values.